conda/recipes/libnvstrings/meta.yaml

```diff
@@ -37,8 +37,10 @@ requirements:
     - flatbuffers
     - boost-cpp
     - dlpack
+    - librdkafka 1.2.2
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - librdkafka 1.2.2
```

Comment:
nvstrings shouldn't require librdkafka

Reply:
I agree ... I very much dislike doing this. I attempted to use target_link_directories() and target_include_directories() to avoid this but I still received CI errors when building.

Reply:
What do you think I should do the remedy this given I have already tried to above process?

Reply:
Can you push the changes with target_include_directories so I can see the CI logs and advise?

Reply:
@kkraus14 just removed librdkafka from the libnvstrings recipe. Give it about 1.5 hours and the CI errors should be good for you to take a look at. Thanks for your help!